contracts/Rentals.sol

```diff
@@ -84,6 +76,12 @@ contract Rentals is NonceVerifiable, NativeMetaTransaction, IERC721Receiver {
         bytes signature;
     }
 
+    struct Rental {
+        address lessor;
+        address tenant;
+        uint256 ending;
```

Comment:
maybe we can use `endDate` to follow the same standard we use in the DB structure, subgraphs and UI